src/elexmodel/handlers/data/Estimandizer.py

```diff
@@ -11,12 +11,15 @@ class Estimandizer:
 
     def add_estimand_results(self, data_df, estimands, historical):
         columns_to_return = []
+        turnout_col = f"{RESULTS_PREFIX}turnout"
+
         for estimand in estimands:
             results_col = f"{RESULTS_PREFIX}{estimand}"
+            additional_columns_added = []
             if results_col not in data_df.columns:
                 # will raise a KeyError if a function with the same name as `estimand` doesn't exist
                 try:
-                    data_df = globals()[estimand](data_df, RESULTS_PREFIX)
+                    data_df, additional_columns_added = globals()[estimand](data_df, RESULTS_PREFIX)
```

Comment:
Sorry, I'm not sure I understand why some estimandized estimands return `additional_columns_added` and some don't 🤔

Reply:
Ah, this is in preparation for margin. We'll be adding both normalized and un-normalized margin to the dataframe. So this just gives us the option to do it.